source/agora/script/Script.d

```diff
@@ -434,7 +434,7 @@ unittest
 *******************************************************************************/
 
 version (unittest)
-public Script makeScript (in ubyte[] opcodes) pure nothrow @safe @nogc
+public Script makeScript (return const ubyte[] opcodes) pure nothrow @safe @nogc
```

Comment:
Why is the `return` keyword used? I don't think it's related to the lifetime of the parameter `opcodes`. Could you explain?

Reply:
It is related to the lifetime. It means that `opcodes` will be part of what's returned by `makeScript`, i.e., part of the `Script` instance.